recipes/boost/all/conanfile.py

```diff
@@ -251,7 +251,27 @@ def config_options(self):
     def _configure_options(self):
         return self._dependencies["configure_options"]
 
+    @property
+    def _fPIC(self):
+        try:
+            return self.options.fPIC
+        except ConanException:
+            return False
+
+    @property
+    def _shared(self):
+        try:
+            return self.options.shared
+        except ConanException:
+            return False
```

Comment:
I am curious if there was any motivation against our usual pattern?

```suggestion
        return self.options.get_safe("shared", False)
```

Reply:
Yes, I missed the pattern. Fixed in a bit different way.